twitter_delete.rb

```diff
@@ -86,6 +87,7 @@ def api_call(method, *args)
   tweets = api_call :favorites, count: MAX_LIKES_PER_PAGE, page: page
   tweets_to_unlike += tweets.reject(&method(:too_new?))
 end
+puts "==> Checking likes to API Completed"
```

Comment:
This is printed after checking but, if you want to print this, should be before.